api/api_full.go

```diff
@@ -859,6 +860,19 @@ type FullNode interface {
 	// Note: this API is only available via websocket connections.
 	// This is an EXPERIMENTAL API and may be subject to change.
 	SubscribeActorEventsRaw(ctx context.Context, filter *types.ActorEventFilter) (<-chan *types.ActorEvent, error) //perm:read
+
+	// F3Participate should be called by a miner node to particpate in signing F3 consensus.
+	// The address should be of type ID
+	// F3Participate can only be used through websocket connection
```

Comment:
Why? IIRC, channels work over HTTP as well.

Reply:
I will remove this doc line for now to avoid confusion.